src/main/java/com/Feature1/WSFeaure1/User.java

```diff
@@ -0,0 +1,39 @@
+package com.Feature1.WSFeaure1;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "User")
+public class User {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @NotBlank
+    private  String name;
+
+    public User(int id, @NotBlank String name) {
```

Comment:
@Xeniia-droid You have `@NotBlank` annotation in two places for `name`. What is the difference and what will happen if you remove one of annotations?

Reply:
The first annotation for the class variable, the second for the method parameter. I think that if I delete the annotation for a method parameter, I can set the method parameter to empty or null. In this case, nothing will happen, because the class variable also has an annotation